cmd/provisioner-localpv/app/provisioner.go

```diff
@@ -50,9 +50,9 @@ import (
 )
 
 const (
-	//KeyNode represents the key values used for specifying the Node Affinity
+	//KeyNodeHostname represents the key values used for specifying the Node Affinity
 	// based on the hostname
-	KeyNode = "kubernetes.io/hostname"
```

Comment:
there seems to be one KeyNode in kubernetes.go of PV.. would it make sense to use it?

Reply:
Need to refactor these kubernetes labels a bit. the current package where they are in also may not be the right place. it should ideally get to a node package. Will also take this up along with the node details refactoring.